admin-ui/app/redux/sagas/AuthSaga.js

```diff
@@ -29,6 +24,7 @@ function* getOAuth2ConfigWorker({ payload }) {
   try {
     const token = !payload?.access_token ? yield* getApiTokenWithDefaultScopes() : payload.access_token
     const response = yield call(fetchServerConfiguration, token)
+    localStorage.setItem('postLogoutRedirectUri', response.postLogoutRedirectUri)
```

Comment:
why we are using localstorage?

Reply:
if user-info is not found, then in that case we use `postLogoutRedirectUri` to end session. Here at that moment we dont have response of `/config` so the use of `localStorage`
![Screenshot 2023-09-13 at 12 17 12 PM](https://github.com/GluuFederation/flex/assets/64732692/345600de-4177-44a2-9686-4412c9769a48)